en/components/grid-toolbar.md

```diff
@@ -6,7 +6,7 @@ _keywords: Design Systems, Design Systems UX, UI kit, Sketch, Ignite UI for Angu
 
 # Grid Toolbar
 
-Use the Grid Toolbar to define actions that affect the appearance and behavior of the whole grid. By default the Toolbar will contain actions for showing/hiding and pinning/unpinning of columns, as well as Exporting the Grid content in various formats. The Grid Toolbar is visually identical to the [Ignite UI for Angular Grid Toolbar Feature](https://www.infragistics.com/products/ignite-ui-angular/angular/components/grid/toolbar.html)
+Use the Grid Toolbar to define actions that affect the appearance and behavior of the whole grid. By default, the Toolbar will contain actions for showing/hiding and pinning/unpinning of columns, as well as Exporting the Grid content in various formats. The Grid Toolbar is visually identical to the [Ignite UI for Angular Grid Toolbar Feature](https://www.infragistics.com/products/ignite-ui-angular/angular/components/grid/toolbar.html)
 
 ## Grid Toolbar Demo
 
```

Comment:
@YoannaIvanova I added one more section could you add an image that has a different layout and elements of the toolbar from the default configuration,